vnext/Microsoft.ReactNative/CompositionSwitcher.idl

```diff
@@ -107,6 +107,7 @@ namespace Microsoft.ReactNative.Composition
     void Size(Windows.Foundation.Numerics.Vector2 size);
     void Position(Windows.Foundation.Numerics.Vector2 position);
     Boolean IsVisible { get; set; };
+    void updateColor(Windows.UI.Color color);
```

Comment:
```suggestion
    void Color(Windows.UI.Color color);
```

nit: I can see that `updateColor` is used in IActivityVisual, but I think the general form of these interfaces are property setter/getters.

Reply:
good to know! Switched ActivityIndicator as well to match :)